src/Resources/FilamentUserResource.php

```diff
@@ -105,7 +109,13 @@ public static function table(Table $table): Table
                 ),
                 ...static::insertAfterTableSchema(),
             ])
-            ->actions([EditAction::make(), ViewAction::make()])
+            ->actions([EditAction::make(), ViewAction::make(), Action::make('reset_password')
+            ->action(function ($record) {
+                // Call the method to send reset link
+                return (new PasswordResetController())->sendResetLink(new Request([
+                    'email' => $record->email, // Pass the user's email
+                ]));
+            })])
```

Comment:
I'm fine with adding this, but please make it less prominent by placing it in a grouped action: https://filamentphp.com/docs/3.x/actions/grouping-actions

While we're at it, might as well add the Extend action to the group

Reply:
I can add it afterwards, but basically it's an action to extend expired accounts. It just needs to call the extend() method on the model. There's a bulk action for it already. It should only be displayed if the expiry feature is enabled

Reply:
Sure. The label for the reset password action should also be translated, doesn't seem like it is in your example?